doc/ld-mapping.md

```diff
@@ -67,20 +67,20 @@ You can browse our definitions as Linked Data on the Web:
 
 #### Describing a measurement
 
-Once the meta data is set one can start to convert each row to RDF data. Data Cube Vocabulary is defining a class called `qb:Observation` for this. Every single row in the CSV file (except the header) will be transformed to a `qb:Observation`. Obviously the most important information assigned to such an observation is what we measured. This means every observation contains a measured value, ideally an assigned unit and then all the other constraints which are necessary to uniquely describe this particular measurement. Doing this properly is essential, we re-use the `qb:ComponentProperty` defined above for this.
+Once the meta data is set one can start to convert each row to RDF data. Data Cube Vocabulary is defining a class called `qb:Observation` for this. Every single row in the CSV file (except the header row) will be transformed to a `qb:Observation`. Obviously the most important information assigned to such an observation is what we measured. This means every observation contains a measured value, ideally an assigned unit, and all the other constraints necessary to uniquely describe this particular measurement. Doing this properly is essential; to do so, the `qb:ComponentProperty` described above can be re-used.
 
 ![Normalized base table as CSV](img/basetable.png)
 
-Once this is done we are set and have a basic RDF Data Cube available.
+Once this is done, we are set and have a basic RDF Data Cube available.
 
 Sample Measurement:
 
 * http://environment.data.admin.ch/ubd/28/measurement/neLCF/O3/nodaysmaxhourlymeanabove120/2002
 * http://environment.data.admin.ch/ubd/28/measurement/tiLUG/pb_dustfall/annualmean/1992
 
-#### Normalizing re-occuring enties in columns
+#### Normalizing re-occuring entries in columns
 
-In our original spreadsheet file we re-use the same terms for describing a re-occuring item in a column. This makes it possible to use pivoting tools to quickly filter for a certain value. If we would model that in a relational database this would be modeled as a relation to its own table (normalization). This is what we need for RDF as well and for that reason the data got prepared that way upfront.
+In our original spreadsheet file the same terms were used for describing a re-occurring item in a column. This makes it possible to use pivoting tools to quickly filter for a certain value. If we were to model the data in a relational database this would be modeled as a relation to its own table (normalization). This is what we need for RDF as well and for that reason the data got prepared that way upfront.
```

Comment:
I guess it would be useful to put the requirements regarding the original spreadsheet file at the very beginning of the instructions.